doc/classes/Node3D.xml

```diff
@@ -232,143 +239,159 @@
 			<param index="1" name="id" type="int" />
 			<param index="2" name="transform" type="Transform3D" />
 			<description>
-				Set subgizmo selection for this node in the editor.
+				Selects the [param gizmo]'s subgizmo with the given [param id] and sets its transform. Only works in the editor.
 				[b]Note:[/b] The gizmo object would typically be an instance of [EditorNode3DGizmo], but the argument type is kept generic to avoid creating a dependency on editor classes in [Node3D].
 			</description>
 		</method>
 		<method name="show">
 			<return type="void" />
 			<description>
-				Enables rendering of this node. Changes [member visible] to [code]true[/code].
+				Allows this node to be rendered. Equivalent to setting [member visible] to [code]true[/code]. This is the opposite of [method hide].
 			</description>
 		</method>
 		<method name="to_global" qualifiers="const">
 			<return type="Vector3" />
 			<param index="0" name="local_point" type="Vector3" />
 			<description>
-				Transforms [param local_point] from this node's local space to world space.
+				Returns the [param local_point] converted from this node's local space to global space. This is the opposite of [method to_local].
 			</description>
 		</method>
 		<method name="to_local" qualifiers="const">
 			<return type="Vector3" />
 			<param index="0" name="global_point" type="Vector3" />
 			<description>
-				Transforms [param global_point] from world space to this node's local space.
+				Returns the [param global_point] converted from global space to this node's local space. This is the opposite of [method to_global].
 			</description>
 		</method>
 		<method name="translate">
 			<return type="void" />
 			<param index="0" name="offset" type="Vector3" />
 			<description>
-				Changes the node's position by the given offset [Vector3].
-				Note that the translation [param offset] is affected by the node's scale, so if scaled by e.g. [code](10, 1, 1)[/code], a translation by an offset of [code](2, 0, 0)[/code] would actually add 20 ([code]2 * 10[/code]) to the X coordinate.
+				Adds the given translation [param offset] to the node's [member position], in parent space (relative to the parent). As such, the final translation is multiplied by the node's [member scale], unlike [method translate_object_local].
 			</description>
 		</method>
 		<method name="translate_object_local">
 			<return type="void" />
 			<param index="0" name="offset" type="Vector3" />
 			<description>
-				Changes the node's position by the given offset [Vector3] in local space.
+				Adds the given translation [param offset] to the node's position, in local space (relative to this node). As such, the final translation is relative to the node's facing direction.
```

Comment:
```suggestion
				Adds the given translation [param offset] to the node's position, in object-local space (relative to this node). As such, the final translation is relative to the node's facing direction.
```

Reply:
I gather this means every mention of it should say "object-local" instead